wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/service/LanguageServiceUtils.java

```diff
@@ -35,6 +35,10 @@ public static String removeUriScheme(String uri) {
     return uri.startsWith(FILE_PROJECTS) ? uri.substring("file://".length()) : uri;
   }
 
+  public static String removeFilePrefix(String uri) {
+    return uri.startsWith("file:/projects") ? uri.substring("file:/projects".length()) : uri;
```

Comment:
Where does `file:/project` come from?

Reply:
It is from the workspace symbols when the file is in your project e.g. "file:/projects/test/src/main/java/tester.java"

Reply:
Could you point out the method?

Reply:
In [this part](https://github.com/eclipse/che/blob/5730_java_ls_poc/wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/service/WorkspaceService.java#L165) o.getLocation().getUri() is being set to something like "file:/projects/test/src/main/java/tester.java" when the file is in your workspace